PySDM/initialisation/aerosol_composition/dry_aerosol.py

```diff
@@ -94,13 +93,13 @@ def kappa(self, mass_fractions: dict):
         result = {}
         for st in formulae._choices(surface_tension).keys():
             if st in (surface_tension.Constant.__name__):
-                result[st] = all_soluble_ns * Mv / rho_w
+                result[st] = all_soluble_ns * const.Mv / const.rho_w
```

Comment:
where does this `const` come from?

Reply:
to make it work, it needs to refer to the `constants` property of the very same instance of `Formulae` as used elsewhere in a simulation, hence we likely want to pass `formulae: Formulae` as a new argument of `kappa()`

Reply:
I'm worried that makes the logic circular. Currently we need an aerosol to make the Formulae object. So we can't need a Formulae to make an aerosol too.... 
```
# aerosol creation
aerosol = AerosolBetaCaryophylleneDark()

# formulae creation
formulae_bulk = Formulae(surface_tension='Constant')
formulae_ruehl = Formulae(
    surface_tension='CompressedFilmRuehl',
    constants={
        'RUEHL_nu_org': aerosol.modes[0]['nu_org'],
        'RUEHL_A0': 115e-20 * si.m * si.m,
        'RUEHL_C0': 6e-7,
        'RUEHL_m_sigma': 0.3e17 * si.J / si.m**2,
        'RUEHL_sgm_min': 35 * si.mN / si.m
    }
)```